quirks/password-rules.json

```diff
@@ -200,5 +200,8 @@
     },
     "xfinity.com": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower, upper; required: digit;"
+    },
+    "signin.ea.com": {
+     "password-rules": "minlength: 8; maxlength: 64; required: lower, upper; required: digit;" 
```

Comment:
Can you make the indentation on this line match the other similar lines in the file? After that, I’m happy to merge. :)